python-meh.spec

```diff
@@ -20,48 +18,24 @@ License: GPLv2+
 Group: System Environment/Libraries
 BuildArch: noarch
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
-BuildRequires: python-devel
 BuildRequires: gettext
-BuildRequires: python-setuptools
 BuildRequires: intltool
-BuildRequires: dbus-python
 BuildRequires: libreport-gtk >= %{libreportver}
 BuildRequires: libreport-cli >= %{libreportver}
-BuildRequires: libreport-python >= %{libreportver}
-BuildRequires: python-six
 
-%if 0%{with_python3}
 BuildRequires: python3-devel
 BuildRequires: python3-setuptools
 BuildRequires: python3-dbus
 BuildRequires: libreport-python3 >= %{libreportver}
 BuildRequires: python3-six
-BuildRequires: python3-pocketlint
-%endif
-
-Requires: python
-Requires: dbus-python
-Requires: rpm-python
-Requires: libreport-cli >= %{libreportver}
-Requires: libreport-python >= %{libreportver}
-Requires: python-six
 
-%description
-The python-meh package is a python library for handling, saving, and reporting
+%global _description\
+The python-meh package is a python library for handling, saving, and reporting\
```

Comment:
I think you want to have space before '\' otherwise I guess it will result in `reportingexceptions`.

Reply:
Fixed.